services/network/networkinterface/networkinterface.go

```diff
@@ -126,6 +126,9 @@ func getWssdNetworkInterfaceIPConfig(ipConfig *network.InterfaceIPConfiguration)
 	if ipConfig.Gateway != nil {
 		wssdipconfig.Gateway = *ipConfig.Gateway
 	}
+	if ipConfig.Primary != nil {
```

Comment:
please update getNetworkIpConfig also so that Primary shows up correctly in get calls.